app/renderer/abr-document.js

```diff
@@ -251,6 +252,35 @@ AbrDocument.prototype = {
         return this.cm.doc.setValue("");
     },
 
+    // Get document title suggestion for save-dialog
+    getTitleSuggestion: function() {
+        var data = this.getData();
```

Comment:
Addressed this. I used a for loop to go over the lines instead of `cm.doc.eachLine()`, so that I can break out when I find what I need. But I used `cm.doc.getLine(i)` to get the current line.